tests/bluetooth/shell/src/main.c

```diff
@@ -121,6 +122,11 @@ static void hrs_notify(void)
 
 void main(void)
 {
+	if (IS_ENABLED(CONFIG_USB_UART_CONSOLE)) {
+		usb_enable(NULL);
```

Comment:
Couldn't USB auto enable when console is used? That should enable logging the init sequence as well.

Reply:
No.  The user may want to register a custom handler.  (I do in real apps.)